cppcache/integration-test/fw_helper.hpp

```diff
@@ -107,7 +105,7 @@ BEGIN_TEST.
 #define ASSERT_EQ(x, y) _ASSERT_EQ(x, y, __LINE__, __FILE__)
 
 #define ASSERT_STREQ(x, y)                                            \
-  ASSERT((strcmp(x, y) == 0),                                         \
+  ASSERT(x == y,                                         \
```

Comment:
I would be a little hesitant to change the expected type of x and y from c-string to `std::string`, or really anything with `==` operator overload.

What about `std::strcmp`?

Reply:
Well thing is this ASSERT_STREQ macro is only used in one place within integration-tests, so I'd rather remove this macro and use operator== in-place.
For reference here is the only place where it's used: https://github.com/apache/geode-native/blob/a072c8f69aa59335ab0b134f435f174c97d0ac22/cppcache/integration-test/testSystemProperties.cpp#L61

As you can see it's safe to remove this macro and use operator== from std::string

Reply:
Yes! So that! One less macro!